dl.py

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env python3
```

Comment:
Shouldn't this be `#!/usr/bin/env python3`?

Reply:
Maybe you misread, but that's exactly what it says.

Reply:
@pingiun Sorry, was confused by GitHub's formatting, it wasn't showing the `#`, my mistake